jekyll/_cci2/machine-runner-3-manual-installation.adoc

```diff
@@ -0,0 +1,111 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Machine runner 3.0 manual installation - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Instructions on how to manually start CircleCI's self-hosted machine runner 3.0 on macOS and Linux.
+:icons: font
+:experimental:
+:machine:
+:macOS:
+:linux:
+
+NOTE: Machine runner 3.0 is currently in open preview
+
+This page describes how to manually start CircleCI's machine runner 3.0 on macOS and Linux.
+
+CAUTION: We recommend installing and using CircleCI machine runner 3.0 with xref:install-machine-runner-3-on-macos.adoc[the Homebrew package] on macOS and xref:install-machine-runner-3-on-linux.adoc[the Linux packages] on Linux. The manual method described on this page is an optional alternative.
+
+WARNING: If you follow this manual installation method, CircleCI machine runner 3.0 will not automatically update.
+
+[#prerequisites]
+== Prerequisites
+
+{% include snippets/runner/machine-runner-prereq.adoc %}
+
+[#self-hosted-runner-terms-agreement]
+== Self-hosted runner terms agreement
+
+{% include snippets/runner/terms.adoc %}
+
+[#create-namespace-and-resource-class]
+== 1. Create namespace and resource class
+
+{% include snippets/runner/install-with-cli-steps.adoc %}
+
+[#download-circleci-runner]
+
+[#download-machine-runner]
+== 2. Download the CircleCI machine runner for macOS
```

Comment:
@jrahme-cci I guess this "for macOS" is just an oversight, does it make sense to just remove that part so the section title is just `2. Download the CircleCI machine runner`?

Reply:
It does. I've made the changes for that.